ksp-psql-conf-info-block.yaml

```diff
@@ -0,0 +1,20 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/
+
```

Comment:
Attach STIG URL in # as well, check https://github.com/kubearmor/policy-templates/pull/460/files for reference